components/UI/Map/LeafletMap.tsx

```diff
@@ -174,6 +168,19 @@ function LeafletMap() {
     ? DEFAULT_ZOOM
     : DEFAULT_ZOOM_MOBILE;
 
+  const ogQueryParamAppender = (marker: MarkerData) => {
+    const clearHash = asPath.replace("/#", "");
+    const address = marker.formatted_address;
+    const entry = marker.source.full_text as string;
+    const loc = `${marker.geometry.location.lat},${marker.geometry.location.lng}`;
+    const constructedQuery = `&address=${encodeURIComponent(
+      address
+    )}&entry=${encodeURIComponent(entry)}&loc=${encodeURIComponent(loc)}`;
+    push({
+      hash: `${clearHash}${constructedQuery}`,
+    });
```

Comment:
Karakter sinirini asar miyiz burada? 

Entrynin ilk belli sayida karakterini alsak oluyor mu? 

https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

Reply:
@cdagli entry için bir karakter sınırımız var mı acaba o detayı ben de bilmiyorum. Deneyebiliriz ancak bunun içeriği de aşağıdaki gibi özellikle resim içerisinde önemli bir bilgi veriyor. Bu bilgi kaybedilebilir.

https://user-images.githubusercontent.com/1084392/217431261-9652161c-a4af-49fe-af4b-4e378fc7f448.png

Reply:
Doğru evet o posta girebilmesi hedef aynı şekilde. Drawerın açılması mevzusunu eklemedim henüz. TODO olarak ekliyorum PR'a. Bi üst limit koyarak devam edelim bence de ne olur ne olmaz.